test-data/unit/stubgen.test

```diff
@@ -2618,32 +2616,29 @@ class A(metaclass=abc.ABCMeta):
     @abc.abstractmethod
     def x(self): ...
 
-[case testClassWithNameIncompleteOrOptional]
+[case testClassWithNameIncomplete]
```

Comment:
Testing for `Optional` seemed obsolete, since we don't use `typing.Optional` anymore, so I removed it while I was here.